Resources/Prototypes/Catalog/Fills/Lockers/dressers.yml

```
@@ -34,6 +34,7 @@
       - id: ClothingHeadHatBeretEngineering
       - id: ClothingOuterWinterCE
       - id: ClothingOuterCoatCEFormalDress #SS220-FormalDress
+      - id: ClothingHeadHatBeretCE #SS220-Resprite
```

```suggestion
      - id: ClothingHeadHatBeretCE #SS220-CEBeret
```
всё же стоит поподробнее писать, что за респрайт